pkg/rc/farm.go

```diff
@@ -277,12 +278,17 @@ func (rcf *Farm) failsafe(rcFields []rcstore.RCLockResult) {
 		}
 		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
 	}
+
 	globalReplicaCount := 0
 	for _, rc := range rcFields {
-		if rc.ReplicasDesired > 0 {
+		if rc.ReplicasDesired == nil {
+			panic(fmt.Sprintf("RC %s is missing replicas_desired count", rc.ID))
```

Comment:
It seems a bit extreme. Then again, it does match up with all other `panic`s in this function, and it's not like this function has any other mechanism (since it doesn't return anything).

I wondered about this because the other checks would only trigger as a result of the state of all RCs as a whole, whereas this check will trigger on just a single RC. Thus I could imagine a scenario where a single bad RC would stop all the other RCs from working. Would this be what we want?